src/main/java/ch/njol/skript/test/runner/SkriptAsyncJUnitTest.java

```diff
@@ -0,0 +1,9 @@
+package ch.njol.skript.test.runner;
+
+/**
+ * All methods in this class marked with @Test will be run on a different thread from the main one.
+ * All tests are still done sequentially.
+ */
+public abstract class SkriptAsyncJUnitTest extends SkriptJUnitTest {
```

Comment:
Can be an interface if no methods are required

Reply:
`SkriptJUnitTest` is an abstract class so this can't be an interface

Reply:
We'll it doesn't need to extend SkriptJUnitTest, but I guess that does make definiting both in a single line becomes redundant.